src/Native/Runtime/unix/UnixContext.cpp

```diff
@@ -257,6 +257,43 @@ static void RegDisplayToUnwindCursor(REGDISPLAY* regDisplay, unw_cursor_t *curso
 #endif // _AMD64_
 }
 
+// Returns the unw_proc_info_t for a given IP.
+bool GetUnwindProcInfo(PCODE ip, unw_proc_info_t *procInfo)
+{
+    int st;
+
+    unw_context_t unwContext;
+    unw_cursor_t cursor;
+
+    st = unw_getcontext(&unwContext);
+    if (st < 0)
+    {
+        return false;
+    }
+
+#ifdef _AMD64_
+    unwContext.data[16] = ip;
```

Comment:
I'll add a note explaining that these 'magic' array indices are temporary and will go away soon. I plan to make a change in the near future after which we will find the LSDA and stuff without initializing an unwind cursor.